### ISSUE TITLE
Add fixes for modernize/six also

### ISSUE THREAD

**Issue by treyhunner** [purplediane/drop2#2]
Shouldn't be too hard to make fixes that also fix things added by modernize (replacing various `from six.moves` lines with the Python 3 version for example).

This might actually be more useful since many packages out there use `six`.

**Comment by WSLUser** [comment]
Keep this project in mind: https://github.com/asottile/pyupgrade.

**Comment by treyhunner** [comment]
Thanks @WSLUser! I hadn't seen that project.

It looks like it doesn't handle `six.moves`, though this library seems to: https://github.com/asottile/reorder_python_imports

Unfortunately that library also reorders imports which is a bit more than I hoped to do.

I may add some links to these in the README and try to complement pyugrade instead of re-implementing it.

**Comment by WSLUser** [comment]
Yeah, that project isn't doing fixtures. Just rewrites code for changes needed. I have found it useful to use after using futurize (or modernize). Also keep in mind the issues they are working on to avoid duplicate effort.
